src/main/js/src/components/UI/RadioButton/RadioButton.module.css

```diff
@@ -14,19 +25,15 @@
 }
 
 .RadioButtonDisabled input {
```

Comment:
The following classes have the same CSS properties, so they can be combined with a comma.

.RadioButtonDisabled input
.RadioButtonDisabled .RadioButtonLabel
.RadioButtonDisabled .RadioButtonExtraLabel